src/sentry/tasks/symbolication.py

```diff
@@ -122,6 +122,7 @@ def get_symbolication_platforms(
         platforms.append(SymbolicatorPlatform.jvm)
     if is_js_event(data, stacktraces):
         platforms.append(SymbolicatorPlatform.js)
+    # TODO @athena: remove comment before merging
     if get_native_symbolication_function(data, stacktraces) is not None:
```

Comment:
Since I added the new platform to native platforms this would return `process_native_stacktraces` which I think is correct. But please help me by verifying.